.golangci.yml

```diff
@@ -9,13 +9,22 @@ linters:
     - revive
     - gosec
     - misspell
-    - scopelint
+    - exportloopref
```

Comment:
WARN [runner] The linter 'scopelint' is deprecated (since v1.39.0) due to: The repository of the linter has been deprecated by the owner.  Replaced by exportloopref.